project-metrics/metrics_service/models.py

```diff
@@ -42,3 +42,29 @@ class MetricResult(Base):
   def __repr__(self) -> Text:
     return "<Metric(name='%s', value='%.3g', computed_at='%s')>" % (
         self.name, self.value, self.computed_at)
+
+
+class TravisState(enum.Enum):
+  """A state of a Travis build or job."""
```

Comment:
Are these values taken from Travis? If so, link to docs

Reply:
It's not "documented" anywhere, it's just defined in the Rails model for [builds](https://github.com/travis-ci/travis-api/blob/master/lib/travis/model/build/states.rb#L25) and [jobs](https://github.com/travis-ci/travis-api/blob/master/lib/travis/model/job/test.rb#L21). Adding comment